staging/src/k8s.io/component-base/config/types.go

```diff
@@ -93,9 +92,10 @@ type LoggingConfiguration struct {
 	// Format Flag specifies the structure of log messages.
 	// default value of format is `text`
 	Format string
-	// Maximum number of seconds between log flushes. Ignored if the
-	// selected logging backend writes log messages without buffering.
-	FlushFrequency time.Duration
+	// The maximum delay between log flushes, specified as a string in the
+	// format understood by time.ParseDuration ("1m", "1h30m"). Ignored if
+	// the selected logging backend writes log messages without buffering.
+	FlushFrequency metav1.Duration
```

Comment:
If we change the type, we will break things that expect the API to work as released in 1.23. We should consider instead adding a new field with the desired type and deprecate the old one, e.g. `FlushFrequencyDuration` as @liggitt suggested.

Reply:
This will be a future improvement to make the configuration more user-friendly. I don't want to keep this bug fix PR simple and not spend more time on it than necessary (mine and of reviewers).

So let's make the documentation match the implementation from 1.23. I'll update the PR.

Reply:
@ehashman: update pushed, please take another look.